v2.0/enable-node-map.md

```diff
@@ -4,4 +4,199 @@ summary: Learn how to enable the node map in the Admin UI.
 toc: false
 ---
 
-*Docs coming soon*
\ No newline at end of file
+The **Node Map** visualizes the geographical configuration of a multi-regional cluster by plotting the node localities on a world map. The **Node Map** also provides real-time cluster metrics, with the ability to drill down to individual nodes to monitor and troubleshoot the cluster health and performance.
+
+This page walks you through the process of setting up and enabling the **Node Map**.
+
+{{site.data.alerts.callout_info}}The <b>Node Map</b> is an <a href="enterprise-licensing.html">enterprise-only</a> feature. However, you can <a href="https://www.cockroachlabs.com/pricing/request-a-license/">request a trial license</a>  to enable the <b>Node Map</b>. {{site.data.alerts.end}}
+
+<div id="toc"></div>
+
+## Setting Up and Enabling the Node Map
```

Comment:
I think we can remove this entire section and just move the scenario details to step 2. That would let us promot all of the steps to h2s.

Reply:
Not sure what you mean here. Let's discuss this in office tomorrow.